apps/main-landing/src/components/hero-section/components/tokens-flow-image.tsx

```diff
@@ -0,0 +1,14 @@
+import Image from 'next/image';
+
+export const TokensFlowImage = () => {
+  return (
+    <Image
+      priority
+      width={1985}
+      height={1240}
+      src="/hero-tokens-flow.png"
+      className="z-0 -mt-[22%] w-full min-w-[500px] md:-mt-[18%] lg:-mt-[15%]"
+      alt=""
+    />
+  );
+};
```

Comment:
same as here
https://github.com/idriss-crypto/browser-extensions/pull/100#discussion_r1826986050

Reply:
there's eslint warning:
```
Arbitrary value classname 'lg:-mt-[15%]' should not start with a dash (-)eslinttailwindcss/enforces-negative-arbitrary-values
```

to get rid of it you should use `lg:mt-[-15%]` so specify `-` sign inside `[]` not outside